src/test/scala/ai/privado/exporter/DataflowExporterValidator.scala

```diff
@@ -18,4 +25,28 @@ trait DataflowExporterValidator {
     alldataflows(Constants.leakages)
   }
 
+  def validateLineNumberForDataflowStep(
+    step: DataFlowSubCategoryPathExcerptModel,
+    expectedLineNumber: Int
+  ): Assertion = {
+    step.lineNumber shouldBe expectedLineNumber
+  }
+
+  def getHeadStepOfDataflow(dataflow: DataFlowSubCategoryModel, sinkId: String): DataFlowSubCategoryPathExcerptModel = {
+    val dataflowForSink = dataflow.sinks.find(sink => sink.id.equals(sinkId))
+    if (dataflowForSink.nonEmpty) {
+      dataflowForSink.get.paths.headOption
+        .getOrElse(DataFlowSubCategoryPathModel("", List.empty[DataFlowSubCategoryPathExcerptModel]))
+        .path
+        .head
+    } else {
+      DataFlowSubCategoryPathExcerptModel("", -1, -1, "", "")
```

Comment:
Instead of playing with some values, we can make this function to return `Option[DataFlowSubCategoryPathExcerptModel]` and enclose 
```
 Try(dataflowForSink.get.paths.head.path.head).toOption
 
 ```
and the else part can return `None`